Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,33 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_environment",
+            category=['society'],
+            prompt="The environment",
+            random=True
+        )
+    )
+ 
+label monika_environment:
+    m 2eub "[player], do you know why I care about the environment?"
+    m 1esa "Well, there are a few reasons."
+    m 3euu "For one, I think it's beautiful."
+    m 1dsd "Vibrant meadows in the spring, chittering forests in the fall..."
+    m 2hub "Breathtaking beaches in the summer, and shimmering snow in the winter."
+    m 7wsu "Sometimes it can be euphoric just to sit down and admire nature, you know?"
```

Comment:
![2](https://user-images.githubusercontent.com/45746485/93007341-546c2980-f568-11ea-80b0-aaa0ca4a92b3.png)

Wide eyes are often reserved for shocked faces, though I see what you were going for. What do you think about this one? I think it fits more with letting your mind drift to something nice.

![image](https://user-images.githubusercontent.com/45746485/93007386-db210680-f568-11ea-8998-0a3de65748ee.png)

Reply:
Yeah, that looks a lot better. I'll adjust that one, thanks for the suggestion!